diff --git a/node_modules/expo-localization/android/build.gradle b/node_modules/expo-localization/android/build.gradle
index 6f160b1..2c87aff 100644
--- a/node_modules/expo-localization/android/build.gradle
+++ b/node_modules/expo-localization/android/build.gradle
@@ -1,11 +1,15 @@
-plugins {
-  id 'com.android.library'
-  id 'expo-module-gradle-plugin'
-}
+apply plugin: 'com.android.library'
 
 group = 'host.exp.exponent'
 version = '16.1.6'
 
+def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
+apply from: expoModulesCorePlugin
+applyKotlinExpoModulesCorePlugin()
+useCoreDependencies()
+useDefaultAndroidSdkVersions()
+useExpoPublishing()
+
 android {
   namespace "expo.modules.localization"
   defaultConfig {
